system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -117,4 +118,36 @@ system_metrics_collector::ProcCpuData LinuxCpuMeasurementNode::makeSingleMeasure
   }
 }
 
+void LinuxCpuMeasurementNode::publishStatistics()
+{
+  if (publisher_ == nullptr) {
+    return;
+  }
+
+  const moving_average_statistics::StatisticData statistic_data = getStatisticsResults();
+  const double data[moving_average_statistics::STATISTICS_DATA_TYPES.size()] = {
```

Comment:
VLA are windows only. Why not using a `std::array` here?

Reply:
This kind of code is no longer part of this PR.